imports/plugins/included/notifications/server/hooks/notification.js

```diff
@@ -16,14 +16,20 @@ const getAdminUserId = () => {
   return false;
 };
 
-const sendNotificationToAdmin = (adminId) => {
+const sendNotificationToAdmin = (adminUserId) => {
```

Comment:
Should we be using adminAccountId here instead?

Reply:
Possibly but we'd have to duplicate the account lookup that's here in the places that call this. It's sort of a tricky mess of account/user references at the moment so I think I'll leave it be.